volume/local/local_test.go

```diff
@@ -124,3 +124,24 @@ func TestCreate(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateName(t *testing.T) {
+	r := &Root{}
+	names := map[string]bool{
+		"/testvol":    false,
+		"thing.d":     true,
+		"hello-world": true,
+		"./hello":     false,
+		".hello":      false,
+	}
+
+	for vol, expected := range names {
```

Comment:
That's a nit but `expected` -> `valid` would make the test a little bit easier to read :stuck_out_tongue_closed_eyes:.